src/time.c

```diff
@@ -96,7 +114,6 @@ int clock_getres( clockid_t clk_id, struct timespec *ts )
     }
     else if ( CLOCK_MONOTONIC_RAW == clk_id ||
               CLOCK_MONOTONIC_RAW_APPROX == clk_id ||
-              CLOCK_PROCESS_CPUTIME_ID == clk_id ||
```

Comment:
We had this value at condition below BTW, missed copy-and-paste issue from the last PR